include/constants/pokemon.h

```diff
@@ -368,6 +368,11 @@ enum MonGender
 };
 #endif //PM_ASM
 
+//Non-Asm Gender Consts
```

Comment:
we already have an enum for this, it's enum MonGender

Reply:
ok well Im making it not an enum bc gender is a u8 and the enum unnecessarily complicates things when compiling